spec/kinde_sdk_spec.rb

```
@@ -1,4 +1,9 @@
 require 'spec_helper'
+require 'spec_helper'
```

RuboCop (1.73)

[warning] Duplicate `require` detected.

(Lint/DuplicateRequire)